### PR TITLE
Add Stripe driver

### PR DESCRIPTION
**Pull request by grkek**
**Description of the change**

Implement the Stripe driver.

### CODE REVIEW

File: drivers/stripe/api.cr [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
...tails : Hash(String, String)? = nil, metadata : Hash(String, String)? = nil, acss_debit : Hash(String, String)? = nil, affirm : Hash(String, String)? = nil, afterpay_clearpay : Hash(String, String)? = nil, alipay : Hash(String, String)? = nil, au_becs_debit : Hash(String, String)? = nil, bacs_debit : Hash(String, String)? = nil, bancontact : Hash(String, String)? = nil, blik : Hash(String, String)? = nil, boleto : Hash(String, String)? = nil, card : Hash(String, String)? = nil, customer_balance : Hash(String, String)? = nil, eps : Hash(String, String)? = nil, fpx : Hash(String, String)? = nil, giropay : Hash(String, String)? = nil, ideal : Hash(String, String)? = nil, interac_present : Hash(String, String)? = nil, klarna : Hash(String, String)? = nil, konbini : Hash(String, String)? = nil, link : Hash(String, String)? = nil, oxxo : Hash(String, String)? = nil, p24 : Hash(String, String)? = nil, paynow : Hash(String, String)? = nil, promptpay : Hash(String, String)? = nil, radar_options : Hash(String, String)? = nil, sepa_debit : Hash(String, String)? = nil, sofort : Hash(String, String)? = nil, us_bank_account : Hash(String, String)? = nil, wechat_pay : Hash(String, String)? = nil)
```
Comment (caspiano) [inline drivers/stripe/api.cr:26]: Can you split this into multiple lines please?
That's a tonne of arguments 😱 

```suggestion
  def add_payment_method(
    type : String,
    billing_details : Hash(String, String)? = nil,
    metadata : Hash(String, String)? = nil,
    acss_debit : Hash(String, String)? = nil,
    affirm : Hash(String, String)? = nil,
    afterpay_clearpay : Hash(String, String)? = nil,
    alipay : Hash(String, String)? = nil,
    au_becs_debit : Hash(String, String)? = nil,
    bacs_debit : Hash(String, String)? = nil,
    bancontact : Hash(String, String)? = nil,
    blik : Hash(String, String)? = nil,
    boleto : Hash(String, String)? = nil,
    card : Hash(String, String)? = nil,
    customer_balance : Hash(String, String)? = nil,
    eps : Hash(String, String)? = nil,
    fpx : Hash(String, String)? = nil,
    giropay : Hash(String, String)? = nil,
    ideal : Hash(String, String)? = nil,
    interac_present : Hash(String, String)? = nil,
    klarna : Hash(String, String)? = nil,
    konbini : Hash(String, String)? = nil,
    link : Hash(String, String)? = nil,
    oxxo : Hash(String, String)? = nil,
    p24 : Hash(String, String)? = nil,
    paynow : Hash(String, String)? = nil,
    promptpay : Hash(String, String)? = nil,
    radar_options : Hash(String, String)? = nil,
    sepa_debit : Hash(String, String)? = nil,
    sofort : Hash(String, String)? = nil,
    us_bank_account : Hash(String, String)? = nil,
    wechat_pay : Hash(String, String)? = nil
  )
```

File: shard.lock [hunk 2]
```diff
@@ -305,6 +305,10 @@ shards:
     git: https://github.com/spider-gazelle/stomp.git
     version: 1.0.1
 
+  stripetease:
+    git: https://github.com/grkek/stripetease.git
+    version: 0.1.0+git.commit.f7693cdda26bad4c58250419b62d1ac7662c8fa7
```
Comment (caspiano) [inline shard.lock:310]: What was the reason for building a stripe lib when [this library](https://github.com/confact/stripe.cr) exists?

**Review by caspiano — CHANGES_REQUESTED** [review]

File: shard.lock (hunk 2 above)
Comment (grkek) [reply shard.lock:310]: It is lacking the functions we need from what I understand

Comment (caspiano) [reply shard.lock:310]: But could you not make a fork and add the functions? I'm into writing a library I just don't know if it's necessary here

Comment (grkek) [reply shard.lock:310]: I have tried the shard and it has some issues, sometimes it crashes on certain values passed to the function, and seems like it hasn't been tested on the never versions of Stripe.

**Review by stakach — APPROVED** [review]
LGTM

**Pull request merged**